routes/product.router.js

```diff
@@ -0,0 +1,65 @@
+const express=require("express");
+const bodyParser=require("body-parser");
+const router=express.Router();
+const { Product } = require("../models/product.model")
+const {extend}=require("lodash")
+
+router.use(bodyParser.json());
```

Comment:
You can call the body-parser before all the routes in the main index file which  will serve the purpose instead of having to call it in every individual router file